### DISCUSSION TITLE
The mechanism for saving conversations is not robust

### DISCUSSION THREAD

**Discussion by CNSeniorious000** [anse-app/anse#38] [General]
Sometimes, a few conversations on the left sidebar can be clicked, but clicking on them will have no effect. I have experienced this several times, but unfortunately, I still cannot replicate it. In fact, I checked the indexedDB and find those conversations not in the `messages` db.

I think this issue may be related to the fact that messages and conversation metadata are saved in two different indexedDBs. This may work fine most of the time, but if the data in these two indexedDBs becomes inconsistent (possibly due to interruption during the storage process), then those "zombie conversations" cannot be deleted unless I manually check and modify indexedDB in DevTools.

**Comment by yzh990918** [comment]
Thanks for the reply, can you create an issue for this issue?

**Comment by CNSeniorious000** [comment]
Sure. I copied it to #40.
